スタンダード第21回演習課題

lqmf3 opened the pull request:
リファクタリングを行いました

lqmf3 committed:
スタンダード第21回演習課題
リファクタリングを行いました

TomooAkamatsu commented on src/main/java/raisetech/StudentManagement/controller/StudentController.java:
```diff
@@ -26,58 +26,62 @@
+//  public StudentDetail showUpdateStudentForm(@PathVariable int id) {
+//    return service.getStudentDetailById(id);
+//  }
+
```
不要なコードであれば消しちゃいましょう。

lqmf3 replied:
動作に影響なかったので削除します！

TomooAkamatsu commented on src/main/java/raisetech/StudentManagement/controller/StudentController.java:
```diff
@@ -26,58 +26,62 @@
+//  @GetMapping("/studentsCourseList")
+//  public  List<StudentsCourses> getStudentsCourseList(){
+//    return service.searchStudentsCoursesList();
+//  }
```
ここも消しましょう。

lqmf3 replied:
こちらも削除しました

lqmf3 committed:
スタンダード第21回演習課題
動作に影響がないので不要な@GetMapping("/updateStudent/{id}")と@GetMapping("/studentsCourseList")のコードを削除
